src/ui/FacetsMobileMode/FacetsMobileMode.ts

```diff
@@ -36,8 +35,9 @@ export class FacetsMobileMode extends Component {
     breakpoint: ComponentOptions.buildNumberOption(),
     /**
      * Whether to display the facets in a modal instead of a pop-up when the end user expands them in mobile responsive mode.
+     * Set this option to `false` in order to keep the old behaviour.
      */
-    isModal: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    isModal: ComponentOptions.buildBooleanOption({ defaultValue: true }),
```

Comment:
Should we just get rid of the old behaviour entirely, we could also get rid of the isModal option then? Or allow users to go back to the old behaviour?

Reply:
For now I'd keep it

Reply:
Keep it as an escape hatch, yes